### ISSUE TITLE
if this package support the latest gradle version would be number 1 in in dart packages since it is still the only deep ar package 

### ISSUE THREAD

**Issue by jack84156** [mtellect/CameraDeepAR#48]

**Comment by filizsenturk** [comment]
I think the same. I came back to pub.dev to check for any updates for this package..

**Comment by jpetro416** [comment]
Is there any alternative? They keep saying just use an older gradle, but that's a terrible and for most impossible to use

**Comment by filizsenturk** [comment]
I couldn't find any solution for that..

**Comment by mtellect** [comment]
> Is there any alternative? They keep saying just use an older gradle, but that's a terrible fix and for most impossible to use

Hi, how is this a terrible fix and how is the plugin impossible to use?
Did you follow the example on the dev branch? https://github.com/mtellect/CameraDeepAR/tree/dev
Did you use the project build.gradle and it didn't work for you? https://github.com/mtellect/CameraDeepAR/blob/dev/example/android/build.gradle
The SDK version for DeepAr used in this plugin is an old SDK (2020) which is well compatible with the gradle version asked of you to use (classpath 'com.android.tools.build:gradle:3.5.0') .

**Comment by filizsenturk** [comment]
The problem is we have to use gradle 3.5.0 but I want to add another package to my project it asks to use higher version for gradle so that this becomes a huge problem.. thank you for upgrading the package for null safety but please make an update for gradle version..

**Comment by jpetro416** [comment]
> > Is there any alternative? They keep saying just use an older gradle, but that's a terrible fix and for most impossible to use
> 
> Hi, how is this a terrible fix and how is the plugin impossible to use? Did you follow the example on the dev branch? https://github.com/mtellect/CameraDeepAR/tree/dev Did you use the project build.gradle and it didn't work for you? https://github.com/mtellect/CameraDeepAR/blob/dev/example/android/build.gradle The SDK version for DeepAr used in this plugin is an old SDK (2020) which is well compatible with the gradle version asked of you to use (classpath 'com.android.tools.build:gradle:3.5.0') .


 Another package in my project uses higher version for gradle and requires 4.2
Most flutter packages are updated to support 4.2. So this package is outdated and not production ready if it can't keep up with the rest of the community.

**Comment by mtellect** [comment]
It doesn't mean that this plugin isn't working.
Comment out the package that's dependent on 4.2 and run it, or you can Fork the repo and make it compatible

**Comment by jpetro416** [comment]
> It doesn't mean that this plugin isn't working. Comment out the package that's dependent on 4.2 and run it, or you can Fork the repo and make it compatible

You're right I could fork it to get 4.2, but I'm concerned about using it in production ready software when gradle 3.5 is 18 pages deep for releases, I'm not sure why the main project can't update
https://github.com/gradle/gradle/releases?page=18
